python/test_tools/rekognition_stubber.py

```diff
@@ -0,0 +1,287 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Stub functions that are used by the Amazon Rekognition unit tests.
+"""
+
+from test_tools.example_stubber import ExampleStubber
+
+
+class RekognitionStubber(ExampleStubber):
+    """
+    A class that implements a variety of stub functions that are used by the
+    Amazon Rekognition unit tests.
+
+    The stubbed functions all expect certain parameters to be passed to them as
+    part of the tests, and will raise errors when the actual parameters differ from
+    the expected.
```

Comment:
Suggestion:
A class that implements stub functions used  by Amazon Rekognition unit tests.
The stubbed functions expect certain parameters to be passed to them as part of the tests, and raise errors if the parameters are not as expected.

Reply:
I like it! Much more concise.